app/assets/stylesheets/companies.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Companies controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: https://sass-lang.com/
```

Comment:
If you don't occupy this file, you can delete it. 

You can use this file to create your own styles related to your view.

Reply:
Done